docs/source/cn/math_ops.py

```diff
@@ -41,3 +41,376 @@
+        >>> out.shape
+        (2, 3)
+
+        # 广播除法
+        >>> input = flow.tensor(np.random.randn(1,1), dtype=flow.float32)
+        >>> other = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> out = flow.div(input,other).numpy()
+        >>> out.shape 
+        (2, 3)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.mul,
+    r"""mul(input, other)
+    
+    计算 `input` 与 `other` 相乘，支持 element-wise、标量和广播形式的乘法。
+    
+    公式为：
+
+    .. math::
+        out = input \\times other
+    
+    示例：
+
+    .. code-block:: python
+
+        >>> import numpy as np
+        >>> import oneflow as flow
+        
+        # element-wise 乘法
+        >>> input = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> other = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> out = flow.mul(input,other).numpy()
+        >>> out.shape
+        (2, 3)
+
+        # 标量乘法
+        >>> input = 5
+        >>> other = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> out = flow.mul(input,other).numpy()
+        >>> out.shape
+        (2, 3)
+
+        # 广播乘法
+        >>> input = flow.tensor(np.random.randn(1,1), dtype=flow.float32)
+        >>> other = flow.tensor(np.random.randn(2,3), dtype=flow.float32)
+        >>> out = flow.mul(input,other).numpy()
+        >>> out.shape 
+        (2, 3)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.reciprocal,
+    r"""reciprocal(x)
+    计算x的倒数，如果x为0，倒数将被设置为0。
+
+    示例：
+
+    .. code-block:: python
+
+        >>> import numpy as np
+        >>> import oneflow as flow
+        
+        >>> x = flow.tensor(np.array([[1, 2, 3], [4, 5, 6]]), dtype=flow.float32)
+        >>> out = flow.reciprocal(x)
+        >>> out.numpy()
+        array([[1.        , 0.5       , 0.33333334],
+               [0.25      , 0.2       , 0.16666667]], dtype=float32)
+    """,
+)
+
+reset_docstr(
+    oneflow.asin,
+    r"""asin(input)
+
+    返回一个新的 tensor 包含 :attr:`input` 中每个元素的反正弦。
+
+    .. math::
+        \\text{out}_{i} = \\sin^{-1}(\\text{input}_{i})
+
+    参数：
+        input (Tensor): 输入张量
+
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> import numpy as np
+        >>> input = flow.tensor(np.array([-0.5,  0.8, 1.0,  -0.8]), dtype=flow.float32)
+        >>> output = flow.asin(input)
+        >>> output.shape
+        oneflow.Size([4])
+        >>> output
+        tensor([-0.5236,  0.9273,  1.5708, -0.9273], dtype=oneflow.float32)
+        >>> input1 = flow.tensor(np.array([[0.8, 1.0], [-0.6, -1.0]]), dtype=flow.float32)
+        >>> output1 = input1.asin()
+        >>> output1.shape
+        oneflow.Size([2, 2])
+        >>> output1
+        tensor([[ 0.9273,  1.5708],
+                [-0.6435, -1.5708]], dtype=oneflow.float32)
+    """,
+)
+
+reset_docstr(
+    oneflow.asinh,
+    r"""asinh()
+    
+    返回一个包含 :attr:`input` 中每个元素的反双曲正弦的新 tensor。
+
+    .. math::
+        \\text{out}_{i} = \\sinh^{-1}(\\text{input}_{i})
```

Comment:
如果使用r字符串，字符是没有转义的，所以不需要两个 `\` ，如果真输入2个（`\\`），在 latex 语法里意味着换行。
```suggestion
        \text{out}_{i} = \sinh^{-1}(\text{input}_{i})
```

不过输入错误是没什么关系的。你这里的问题是，自己的截图中有明显的排版问题，自己难道无法发现的吗？（既没有修正，也没有求助，就放在那里等别人review）